src/configStore.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * Config redux store
+ *
+ * @author Vinh Le <lethanhvinh95@gmail.com>
+ *
+ */
+
+import { createStore, applyMiddleware, compose } from 'redux'
+import { rootReducer } from './reducer'
+import thunk from 'redux-thunk'
+
+// Config redux devtool
+// Redux does not provide types for __REDUX_DEVTOOLS_EXTENSION_COMPOSE__
+declare global {
+	interface Window {
+		__REDUX_DEVTOOLS_EXTENSION_COMPOSE__?: typeof compose
+	}
+}
+
+const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose
+
+const store = createStore(rootReducer, composeEnhancers(applyMiddleware(thunk)))
```

Comment:
consider adding `redux-logger` to middleware (only in development env)
https://github.com/LogRocket/redux-logger

```
import {createLogger} from "redux-logger";

 if (process.env.NODE_ENV !== "production") {
    const logger = createLogger();
    middlewares.push(logger);
  }

```